packages/api/test.js

```diff
@@ -1,8 +1,9 @@
-import * as api from './index';
+let api = require('./dist/index');
```

Comment:
Tests shouldn't rely on the compiled code, but instead the source. Any way we can change this back?

Reply:
IMO this should be testing the compiled code to make sure that it actually behaves correctly. It isn’t automatically run at this point (#1074) but running directly against the source somehow could give us false confidence.

Reply:
Actually.. this file is not even used. So I would vote to kill it off entirely.

Even when we add proper tests to api: it won't really make much sense since it's not really a unit test or a e2e test. Just some weird leftover I think.

Reply:
🔥